pkg/controller/acicontainersoperator.go

```diff
@@ -401,6 +521,22 @@ func (c *Controller) ReadConfigMap(field string) ([]byte, error) {
 	return raw, err
 }
 
+func (c *Controller) WriteConfigMap(field string, data *corev1.ConfigMap) error {
+	rawIn := json.RawMessage(data.Data["spec"])
+	data_byte, err := rawIn.MarshalJSON()
+	if err != nil {
+		log.Error(err)
+		return err
+	}
+	err = ioutil.WriteFile(field, data_byte, 0777)
+	if err != nil {
+		log.Error(err)
+		return err
+	}
+	log.Info("Writing the Config-Map: ",data.Name," in the file")
```

Comment:
There is no write happening after this log statement. Either the log statement should be "Wrote..." or we should move this log statement to before the write is done...